bokehjs/src/lib/core/property_mixins.ts

```diff
@@ -18,6 +18,15 @@ export type Fill = {
   fill_alpha: p.Property<number>
 }
 
+export type Hatch = {
+  hatch_color: p.Property<Color | null>
+  hatch_alpha: p.Property<number>
+  hatch_scale: p.Property<number>
+  hatch_pattern: p.Property<string>
+  hatch_weight: p.Property<number>
+  hatch_extra: p.Property<{[key: string]: p.Instance}>
```

Comment:
`p.Instance` should be replaced with `Texture`. `T` in `p.Property<T>` should never contain any property types.